gradle/libs.versions.toml

```diff
@@ -119,7 +119,7 @@ postgresql = "org.postgresql:postgresql:42.7.4"
 
 prometheus-caffeine = "io.prometheus:simpleclient_caffeine:0.16.0"
 
-shedlock-jdbc = "net.javacrumbs.shedlock:shedlock-provider-jdbc:5.16.0"
+shedlock-jdbc = "net.javacrumbs.shedlock:shedlock-provider-jdbc:6.0.2"
```

Comment:
Vi har denne avhengigheten fordi den benyttes av kafka-biblioteket fra common-java-modules. Burde kanskje oppgradere shedlock til ny major der først for å være på den sikre siden